deploy/examples/bucket-topic.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       uri: http://my-notification-endpoint:8080
     # uri: https://my-notification-endpoint:8443
       disableVerifySSL: true
+      sendCloudEvents: false # supported only in Ceph Quincy (v17) or newer
```

Comment:
Need a comment about this new setting in ceph-object-bucket-notifications.md?

Reply:
added here: c5350327c182ef04acb4edeaa2fd4b651781a5b8